Mathlib/Analysis/SpecialFunctions/Pow/NNReal.lean

```diff
@@ -547,6 +547,12 @@ theorem rpow_mul (x : ℝ≥0∞) (y z : ℝ) : x ^ (y * z) = (x ^ y) ^ z := by
       simp [coe_rpow_of_ne_zero h, coe_rpow_of_ne_zero this, NNReal.rpow_mul]
 #align ennreal.rpow_mul ENNReal.rpow_mul
 
+theorem rpow_inv_rpow_self {y : ℝ} (hy : y ≠ 0) (x : ℝ≥0∞) : (x ^ y) ^ (1 / y) = x := by
```

Comment:
```suggestion
theorem rpow_inv_rpow {y : ℝ} (hy : y ≠ 0) (x : ℝ≥0∞) : (x ^ y) ^ y⁻¹ = x := by
```
to match `Real.rpow_inv_rpow` (yes, I know, `NNReal.rpow_inv_rpow_self` is misnamed and misstated, feel free to fix it if you want). Same below

Reply:
Under NNReal there is
```
@[simp] lemma rpow_rpow_inv {y : ℝ} (hy : y ≠ 0) (x : ℝ≥0) : (x ^ y) ^ y⁻¹ = x := by
@[simp] lemma rpow_inv_rpow {y : ℝ} (hy : y ≠ 0) (x : ℝ≥0) : (x ^ y⁻¹) ^ y = x := by
-- as well as (proved invoking the above lemmas via simp) --
theorem rpow_inv_rpow_self {y : ℝ} (hy : y ≠ 0) (x : ℝ≥0) : (x ^ y) ^ (1 / y) = x := by
theorem rpow_self_rpow_inv {y : ℝ} (hy : y ≠ 0) (x : ℝ≥0) : (x ^ (1 / y)) ^ y = x := by
```
The only difference in the names is I guess justified by the difference is in the notation for the inverse. Just to confirm, your recommendation is to just keep the first pair, rather than extend the symmetry to Real and ENNReal?

Reply:
Yes, keep only the first pair